src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -1364,8 +1362,6 @@ public void testUpdateGenotypeAfterSubsetting(final int ploidy,
                                                   final List<Allele> allelesToUse,
```

Comment:
Might want to rename this test method to reflect the new method name -- eg., `testMakeGenotypeCall`